infoTree/out/url100.txt

```diff
@@ -0,0 +1,17 @@
+{
+	"title": "Gasoline",
+	"heading": "Hazardous ? no charge",
```

Comment:
Yes this is what all apostrophe, hyphens and misc characters appear as. I'm still having trouble figuring this out, and still haven't done my reading on character encodings to fully understand what the fix for this would be